src/transformers/models/xlm_roberta_xl/modeling_xlm_roberta_xl.py

```diff
@@ -47,7 +47,7 @@
 
 logger = logging.get_logger(__name__)
 
-_CHECKPOINT_FOR_DOC = "xlm-roberta-xlarge"
+_CHECKPOINT_FOR_DOC = "facebook/xlm-roberta-xl"
 _CONFIG_FOR_DOC = "XLMRobertaXLConfig"
 
 XLM_ROBERTA_XL_PRETRAINED_MODEL_ARCHIVE_LIST = [
```

Comment:
As an aside, this `XLM_ROBERTA_XL_PRETRAINED_MODEL_ARCHIVE_LIST` list already has the valid checkpoint name, and the old invalid checkpoint is not in the list. If the CI tests checked whether `_CHECKPOINT_FOR_DOC in *_ARCHIVE_LIST` for each model then this kind of typo would be prevented.

Reply:
Feel free to add a check in this or a follow up PR! I'm not 100% sure this will hold for all the models in the library - there might be some special cases when a different checkpoint is used for the sake of a doc example.

Reply:
Yeah I wasn't sure if I should change this specific model's checks since all 100+ models follow a coding template.

I actually went through every `transformers` text model over the last week (to add them to ONNX TurnkeyML and ONNX Model Zoo) and they all had correct checkpoint references except for this one. I would be interested in adding the check described above in a future PR if this kind of problem became more prevalent.

Reply:
Sounds good. Thanks for all of your work on this!